packages/front-end/components/Modal.tsx

```diff
@@ -173,7 +173,7 @@ const Modal: FC<ModalProps> = ({
           children
         )}
       </div>
-      {submit || (close && includeCloseCta) ? (
+      {submit || secondaryCTA || (close && includeCloseCta) ? (
```

Comment:
My only remark would have been to put this into a purposely named variable like `shouldShowBottom` or something similar.